### ISSUE TITLE
tqdm library is not working as expected in Demo.ipynb

### ISSUE THREAD

**Issue by paulavidela** [EL-BID/urbanpy#12]
I tried running _Demo.ipynb_ and I was not able.    
The following function:
`tqdm_notebook().pandas()`
generates this error: 
_Widget Javascript not detected.  It may not be installed or enabled properly._

I have checked and Ipywidgets was installed and enabled for Jupyter.

If I deleted tqdm: 
`
from tqdm import tqdm_notebook
tqdm_notebook().pandas()
`
and I changed progress_apply to apply: 
`
distance_duration = hex_lima.apply(
    lambda row: up.routing.osrm_route(
        origin=row.geometry.centroid, 
        destination = fs.iloc[row['nearest_food_facility_ix']]['geometry']
    ),
    result_type='expand',
    axis=1,
)`

_Demo.ipynb_ worked perfectly. 

Should I set something different in order to use tqdm? Thanks!

**Comment by Claudio9701** [comment]
Hola Paula! Thanks for reporting this issue 🤗

I will try to replicate this error and try to find a solution. It would be helpful if you could send me your environment settings: os, python version, if you're using conda or pip to install packages, and other info you think could be useful.

**Comment by paulavidela** [comment]
Hola Claudio! Thanks :) 
OS: Ubuntu V20.04.3 LTS (Focal Fossa) 
Python version: 3.8.10
I used pip to install UrbanPy

**Comment by Claudio9701** [comment]
Hola de nuevo Paula!

I've tested the error you mentioned using a docker image with the following Dockerfile specs to replicate your environment settings:

```dockerfile
# syntax=docker/dockerfile:1
FROM ubuntu:20.04
RUN apt update
RUN apt install software-properties-common -y
RUN add-apt-repository ppa:deadsnakes/ppa -y
RUN apt install python3.8 python3-pip git -y
RUN pip install --upgrade pip
RUN pip install urbanpy jupyterlab
RUN git clone https://github.com/EL-BID/urbanpy.git
WORKDIR /urbanpy/notebooks
CMD ["jupyter", "lab",  "--ip", "0.0.0.0", "--port", "8888", "--no-browser", "--allow-root"]
EXPOSE 8888
``` 

When I try to run `Demo.iypnb` the following error occurs 😔:
![image](https://user-images.githubusercontent.com/24416566/169911543-134ad74c-125d-4f02-a80e-a5f033000a7d.png)

To solve this problem I followed these steps:

1. Install nodejs with apt and ipywidgets with pip in the terminal:

```sh
$ sudo apt install nodejs
$ pip install ipywidgets
```

1. Then I closed the notebook, shutdown the kernel, and refreshed the webpage in my browser 

1. Finally I opened the notebook and tested that the progress bar is showing correctly 🎉
![image](https://user-images.githubusercontent.com/24416566/169914863-a486a0f2-1312-4355-8178-dceb5dc2b14a.png)
 
I hope this solution alternative helps you. Let me know if I can support you in anything else 🙌🏽.

**Comment by Claudio9701** [comment]
Addressed with commit: 3c773e1739e86b3c0ff1c5c45d1b6c2276a0a6c3

Importing tqdm from tqdm.auto as recommended in [tqdm repo](https://github.com/tqdm/tqdm#ipython-jupyter-integration)
